### DISCUSSION TITLE
WALC v0.3.2

### DISCUSSION THREAD

**Discussion by daffamumtaz2361** [cstayyab/WALC#233] [General]
## Bug fixes:
- Fix whatsapp integration
- Fix offline page sometimes stuck retrying

<hr /><em>This discussion was created from the release <a href='https://github.com/WAClient/WALC/releases/tag/v0.3.2'>WALC v0.3.2</a>.</em>

**Comment by albakhtari** [comment]
# NOTE for Arch users who installed the `walc` package from the AUR:

Currently there's an error when starting WALC which breaks WhatsApp integration.
A quick workaround is to navigate to `/opt/WALC` and run `sudo mkdir -m 755 .wwebjs_cache && sudo chmod 755 -R ./.wwebjs_cache`

The problem is that it needs to be run at every initiation of WALC.
I'm not sure on how to approach the issue, and don't have the time to go through the debugging process at the moment. But I'm open to suggestions.

You can install the [`walc-appimage`](https://aur.archlinux.org/packages/walc-appimage) package (not maintained by me) instead, that's working as intended.

**Reply by daffamumtaz2361** [reply]
Is this still happening to the new version?
